PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Don't just launch into running the diagnostics. Write some text talking about what you are about to do and what you're hoping to find out.  What is a problem if this test fails?</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Added the following lines explaining the reason behind using the diagnostic, what we expect to find out and how to proceed if the test fails:</p><p><br></p><p>We use the Gelman, Rubin, and Brooks Diagnostic to check whether our chains have converged. Note that we require multiple chains to use this diagnostic which analyses the difference between these multiple chains.&nbsp;</p><p><br></p><p>We expect the chains to have converged. This is because we have taken sufficient number of iterations (1500) for the NUTS sampler. However, in case the test fails, then we will have to take a larger number of iterations, resulting in longer computation time.</p>

Reply:
As I mentioned in one of the earlier comments (increasing the value of `q`), I noticed that the chains did not converge.

Reply:
Good addition!